gto/api.py

```diff
@@ -55,6 +55,7 @@ def annotate(
     must_exist: bool = False,
     labels: List[str] = None,
     description: str = "",
+    requirements: List[str] = None,
```

Comment:
typer support for dict is non-existing (as far as I can tell from quick look). Should we parse list from CLI but convert to key-value internally?